src/sbl_filing_api/entities/models/dao.py

```diff
@@ -90,6 +90,7 @@ class ContactInfoDAO(Base):
     hq_address_zip: Mapped[str] = mapped_column(String(5))
     email: Mapped[str]
     phone_number: Mapped[str]
+    extension: Mapped[str] = mapped_column(String(254), nullable=True)
```

Comment:
Could you make this `phone_ext`. I think that's what we agreed to and the last meeting, and I _think_ what @shindigira is coding to on https://github.com/cfpb/sbl-frontend/pull/961.

Reply:
Done